msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/MessageForumStatisticsBean.java

```diff
@@ -1971,22 +2167,24 @@ private String getExternalParameterByKey(String parameterId)
 
 	public String getButtonUserName() {
 		String userName;
-		String firstName;
-		String lastName;
-		int firstIndex = 0;
-		int secondIndex = 0;
-		if (ServerConfigurationService.getBoolean("msg.displayEid", true)) {
```

Comment:
Now that I'm looking at this again, both conditions are trying to parse the name into firstName + " " + lastName; the sakai.property was used to determine how to parse the 'selectedSiteUser' String to achieve this. The new code doesn't parse this property at all, and instead uses User.getDisplayName(). I'd argue that User.getDisplayName() is more appropriate than parsing the selectedSiteUser to generate firstName + " " + lastName (Ie. this considers contextualUserDisplayService, the DisplayAdvisoryUDP, etc.). So I think we're good here